src/app/tasks/create-task/create-task.component.ts

```diff
@@ -1,22 +1,27 @@
 import { Component, EventEmitter, Output } from '@angular/core';
-import { Task } from '../task.model';
+import { Task, TaskStatus, TaskType } from '../task.model';
 import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
 import { TaskService } from '../task.service';
 import { Validators } from '@angular/forms';
+import { KeyValuePipe } from '@angular/common';
 
 @Component({
   selector: 'app-create-task',
   standalone: true,
   imports: [
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    KeyValuePipe
   ],
   templateUrl: './create-task.component.html',
   styleUrl: './create-task.component.scss'
 })
 export class CreateTaskComponent {
-  
+ 
   @Output() taskCreatedEvent = new EventEmitter<Task>();
 
+  taskTypes : {key: string, value: string}[] = [];
```

Comment:
If you're going to use this type of object in multiple places it's better to create a new model for it